examples/text_labels_and_annotations/mathtext_examples.py

```diff
@@ -68,8 +68,8 @@ def doall():
     plt.gca().set_ylim(0., 1.)
     plt.gca().set_title("Matplotlib's math rendering engine",
                         color=mpl_grey_rvb, fontsize=14, weight='bold')
-    plt.gca().set_xticklabels("", visible=False)
-    plt.gca().set_yticklabels("", visible=False)
```

Comment:
What is visible=False doing here?/is it okay that it's dropped?

Reply:
It sets all of the 0 created ticklabels to invisible, i.e. does nothing.